AppLovin MAX Demo App - Swift/AppLovin MAX Demo App - Swift/MAX/Native Ads/ALMAXTemplateNativeAdViewController.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  ALMAXTemplateNativeAdViewController.swift
+//  AppLovin MAX Demo App - Swift
+//
+//  Created by Billy Hu on 1/20/22.
+//  Copyright © 2022 AppLovin. All rights reserved.
+//
+
+import UIKit
+import Adjust
+import AppLovinSDK
+
+class ALMAXTemplateNativeAdViewController: ALBaseAdViewController, MAAdRevenueDelegate
+{
+    @IBOutlet weak var nativeAdContainerView: UIView!
+    
+    private var nativeAdLoader: MANativeAdLoader!
+    private var nativeAd: MAAd?
+    private var nativeAdView: UIView?
```

Comment:
nit but follow ordering of ObjC:
```suggestion
    private var nativeAdView: UIView?
    private var nativeAd: MAAd?
```